### PR TITLE
SERVICE: Object Relations Changed To Id In DTOs

### LINKED ISSUE #23: SERVICE: There Is An Error In The DTOs
image

`public UserForCreationDto Author { get; set; }`
should be ->>  `public long AuthorId {get;set;}` instead of 
and others too.

Assigned to @Temurkhann

### PR DESCRIPTION
**Pull request by Temurkhann**
closes #23

### CODE REVIEW

File: src/ZaminEducation.Service/DTOs/CoursesDtos/CourseForCreationDto.cs [hunk 1]
```diff
@@ -13,13 +13,13 @@ public class CourseForCreationDto
         [MaxLength(100)]
         public string Description { get; set; }
 
-        public UserForCreationDto Author { get; set; }
+        public long AuthorId { get; set; }
 
         public int ViewCount { get; set; }
```
Comment (Dastanbayev) [inline src/ZaminEducation.Service/DTOs/CoursesDtos/CourseForCreationDto.cs:18]: `public ViewCount { get; set; }` menimcha kerak emas 🤔, chunki Course Create qilyotganda hali hech narsa Ko'rilmagan bo'ladi

File: src/ZaminEducation.Service/DTOs/UsersDtos/UserForCreationDto.cs [hunk 2]
```diff
@@ -18,15 +18,15 @@ public class UserForCreationDto
         [Required, MinLength(5)]
         public string Password { get; set; }
 
-        public AddressForCreationDto Address { get; set; }
+        public long Address { get; set; }
```
Comment (Dastanbayev) [inline src/ZaminEducation.Service/DTOs/UsersDtos/UserForCreationDto.cs:21]: Change property name `Address` to `AddressId`

File: src/ZaminEducation.Service/DTOs/UsersDtos/UserForCreationDto.cs [hunk 3]
```diff
@@ -18,15 +18,15 @@ public class UserForCreationDto
...
 
         [MaxLength(200)]
         public string Bio { get; set; }
 
         [Required]
         public Gender Gender { get; set; }
 
-        public AttachmentForCreationDto Image { get; set; }
+        public long Image { get; set; }
```
Comment (Dastanbayev) [inline src/ZaminEducation.Service/DTOs/UsersDtos/UserForCreationDto.cs:29]: Change property name `Image` to `ImageId`

File: src/ZaminEducation.Service/DTOs/UsersDtos/RegionForCreationDto.cs [hunk 4]
```diff
@@ -7,6 +7,6 @@ public class RegionForCreationDto
         [MaxLength(50), Required]
         public string Name { get; set; }
 
-        public RegionForCreationDto Parent { get; set; }
+        public long Parent { get; set; }
```
Comment (Dastanbayev) [inline src/ZaminEducation.Service/DTOs/UsersDtos/RegionForCreationDto.cs:10]: Change property name `Parent` to `ParentId`

**Review by MrTukhtaboev — APPROVED** [review]

File: src/ZaminEducation.Service/DTOs/QuizzesDtos/QuestionAnswerForCreationDto.cs [hunk 5]
```diff
@@ -2,12 +2,12 @@
...
-        public QuizContentForCreationDto Content { get; set; }
+        public long ContentId { get; set; }
 
         public bool IsCorrect { get; set; }
 
-        public QuizForCreationDto Quiz { get; set; }
+        public long QuizId { get; set; }
 
-        public QuizContentForCreationDto Question { get; set; }
+        public long Question { get; set; }
```
Comment (Dastanbayev) [inline src/ZaminEducation.Service/DTOs/QuizzesDtos/QuestionAnswerForCreationDto.cs:11]: Change property name `Question` to `QuestionId`

**Pull request merged**